### PR TITLE
Airports lat long testing

### PR DESCRIPTION
**Pull request by okie-creator**
Pull request asked from meeting. 

Given Time and location goes in the general direction between DFW and OKC and does all the API pull between then

### CODE REVIEW

File: AirportsLatLongConverter.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import airportsdata
```
Comment (n-grounds) [inline AirportsLatLongConverter.py:1]: where does this module come from?  does it have to be installed?  is it built-in to all Python installations?

File: GetNOTAM.py [hunk 2]
```diff
@@ -1,20 +1,107 @@
...
+    location = input("Enter airport: ")
+    finalLocation = input("Enter ending airport: ")
+
+    airLocation = AirportsLatLongConverter.get_lat_and_lon(location)
+    finalAirLocation = AirportsLatLongConverter.get_lat_and_lon(finalLocation)
+
+
+    effectiveStartDate = ZuluConverter.convert_cst_to_zulu(effectiveStartDate)
+    effectiveEndDate = ZuluConverter.convert_cst_to_zulu(effectiveEndDate)
+    headers = {'client_id': credentials.clientID, 'client_secret': credentials.clientSecret}
```
Comment (n-grounds) [inline GetNOTAM.py:33]: why is this variable defined here and never used?

File: AirportsLatLongConverter.py (hunk 1 above)
Comment (okie-creator) [reply AirportsLatLongConverter.py:1]: It's an installation that Gabe found. I should include it in the read me

**Comment by n-grounds** [comment]
I find that for large Pull Requests like this it is super helpful to reviewers to take a screenshot of the successfully executed code result showing the "UI / display / output" given the user's input, etc.

File: GetNOTAM.py [hunk 3]
```diff
@@ -1,20 +1,107 @@
...
+import os
+import ZuluConverter
+import AirportsLatLongConverter
+
+
+
+# Base URL for the NOTAM API
+
+
+def merge_two_dicts(x, y):
```
Comment (n-grounds) [inline GetNOTAM.py:15]: is this function used anywhere?  if so I've missed it.  if not, why do we have it?  if we need it, can we write unit tests for it?

File: GetNOTAM.py (hunk 2 above)
Comment (okie-creator) [reply GetNOTAM.py:33]: That's a mistake from breaking the class into three functions. I just deleted it.

File: GetNOTAM.py (hunk 3 above)
Comment (okie-creator) [reply GetNOTAM.py:15]: The function isn't necessary. I just deleted it.

File: GetNOTAM.py [hunk 4]
```diff
@@ -1,20 +1,107 @@
...
+    z = x.copy()   # start with keys and values of x
+    z.update(y)    # modifies z with keys and values of y
+    return z
+
+def startNotam():
```
Comment (n-grounds) [inline GetNOTAM.py:20]: in Python if you put a string immediately after the `def` line of a function then Python IDEs will treat that as the documentation of the function, which is super helpful.  I highly recommend writing at least a basic description of the function as one of these "docstrings", especially if the function has parameters that need to be documented as to what kind of data they support/expect, etc.

File: AirportsLatLongConverter.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import airportsdata
+
+#airports = airportsdata.load('IATA')
+
+# Should we add a name to abbreviation converter?
+# How will users input their airport?
+
+# Get lat and lon
+def get_lat_and_lon(input_airport):
```
Comment (n-grounds) [inline AirportsLatLongConverter.py:9]: documentation string after this function definition explaining what it expects as input and what kind of return it gives back (e.g., return value will be a two-element list of float values representing latitude and longitude)

Comment (n-grounds) [inline AirportsLatLongConverter.py:9]: unit tests for this function?  what values are expected given a valid airport like `KOKC` or `KLAX`?  what behavior is expected if we give invalid airport names like `DNE` or `None`?

File: ZuluConverter.py [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    return zulu_datetime_formatted
+
+
+#HST to Zulu
+def convert_hst_to_zulu(hst_datatime):
+    hst_tz = timezone('US/Hawaii')     # Hawaii Standard Time
+    zulu_tz = timezone('UTC')   # Universal Time (Zulu)
+    date_format = "%Y-%m-%d %H:%M:%S"
+    hst = datetime.strptime(hst_datatime, date_format)
+    localized_hst = hst_tz.localize(hst)
```
Comment (Gabrieldowen) [inline ZuluConverter.py:80]: maybe in the future we could condense all these "convert blank to Zulu" into one function. We could use the timezone as a variable to pass in

File: AirportsLatLongConverter.py (hunk 5 above)
Comment (okie-creator) [reply AirportsLatLongConverter.py:9]: It runs to error in all cases that aren't in IATA format ('DFW', 'OKC', etc). I'll make a note to return a string statement if the format is incorrect.

File: ZuluConverter.py (hunk 6 above)
Comment (okie-creator) [reply ZuluConverter.py:80]: Sounds good. I can add it as an issue

File: AirportsLatLongConverter.py (hunk 1 above)
Comment (okie-creator) [reply AirportsLatLongConverter.py:1]: added the instructions to install the airportdata to the Readme

**Comment by okie-creator** [comment]
Screenshot 2024-02-16 123334
Screenshot 2024-02-16 123252
Showing the current implantation running. Numbers are the total Notam of each buildNotam run then the lat and long

File: AirportsLatLongConverter.py (hunk 5 above)
Comment (okie-creator) [reply AirportsLatLongConverter.py:9]: Sir, I'm going to add this as an issue to work on to return a string.

**Comment by Gabrieldowen** [comment]
What file did you run for this?

**Comment by okie-creator** [comment]
runNotam() function in getNotam.py
Screenshot 2024-02-16 123905

**Pull request merged**